src/python/dxpy/workflow_builder.py

```diff
@@ -502,7 +576,7 @@ def _build_global_workflow(json_spec, args):
 
         # Create a new global workflow version on the platform
         global_workflow_id = dxpy.api.global_workflow_new(gwf_final_json)["id"]
```

Comment:
Have to look in more detail at what happens inside that global_workflow_new() in order to review this.